src/main/flight/interpolated_setpoint.c

```diff
@@ -58,9 +69,42 @@ FAST_CODE_NOINLINE float interpolatedSpApply(int axis, bool newRcFrame, ffInterp
         const float rxInterval = currentRxRefreshRate * 1e-6f;
         const float rxRate = 1.0f / rxInterval;
 
-        const float setpointSpeed = (rawSetpoint - prevRawSetpoint[axis]) * rxRate;
-        const float setpointAcceleration = (setpointSpeed - prevSetpointSpeed[axis]) * pidGetDT();
+        float setpointSpeed = (rawSetpoint - prevRawSetpoint[axis]) * rxRate;
+        float setpointAcceleration = setpointSpeed - prevSetpointSpeed[axis];
+        const float holdSteps = 2.0f;
```

Comment:
Yes they could be integers, for sure.  I am not sure however if for code efficiency it is better in this FAST block that they are floats.  Happy to change if it would be better.  @etracer65 you thoughts?

Reply:
@ctzsnooze: No, ints will be calculated faster.

Reply:
Changed all to uint8_t, thanks